### PR TITLE
Update `tiup bench` with more tpch flags

### PR DESCRIPTION
**Pull request by JaySon-Huang**
### What problem does this PR solve? 
* Port latest changes in [pingcap/go-tpc](https://github.com/pingcap/go-tpc) to make it more friendly to run a benchmark on TiDB cluster with TiFlash.

### What is changed and how it works?
* Port latest files in `pingcap/go-tpc` · `cmd/go-tpc` to `components/bench`
* Add Makefile for components/bench

### Check List 

Tests 

 - Manual test (add detailed scripts or steps below)

Code changes

 - None

Side effects

 - Breaking backward compatibility

Related changes

 - Need to release a new version of `tiup bench`

### CODE REVIEW

File: components/bench/misc.go [hunk 1]
```diff
@@ -38,7 +44,7 @@ func execute(ctx context.Context, w workload.Workloader, action string, index in
 	switch action {
 	case "prepare":
 		// Do cleanup only if dropData is set and not generate csv data.
-		if dropData && !w.DataGen() {
```
Comment (JaySon-Huang) [inline components/bench/misc.go:41]: Not sure whether we need this or not, please check it carefully.

File: components/bench/misc.go [hunk 2]
```diff
@@ -106,12 +112,8 @@ func executeWorkload(ctx context.Context, w workload.Workloader, action string)
 	wg.Wait()
 
 	if action == "prepare" {
-		if !w.DataGen() {
```
Comment (JaySon-Huang) [inline components/bench/misc.go:109]: Not sure whether we need this or not, please check it carefully.

**Comment by mahjonp** [comment]
LGTM

**Comment by JaySon-Huang** [comment]
@lucklove PTAL

**Pull request merged**